core/domain/config_domain.py

```diff
@@ -224,7 +226,6 @@ def update_moderator_ids():
     [])
 
 
-# TODO(milit): Check the language codes are valid
 LANGUAGES_SUPPORTED = ConfigProperty(
```

Comment:
I didn't understand your previous comment well, that's why I didn't change it to feconf. Done now!